### PR TITLE
ELEMENTS-885: fix nuxeo-selectivity multiple and single inputs height

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

File: ui/widgets/nuxeo-selectivity.js [hunk 1]
```diff
@@ -6962,22 +6957,18 @@ input[type='text'].selectivity-multiple-input:focus {
...
-  padding: 5px;
+  padding: 5px 5px 0px 5px;
   position: relative;
   -webkit-box-sizing: content-box;
   box-sizing: content-box;
 }
 
 .selectivity-single-select-input {
   opacity: 0;
+  display: none;
```
Comment (nmpcunha) [inline ui/widgets/nuxeo-selectivity.js]: Do we really need this? I'm not sure about its purpose. Could you please provide further details?

File: ui/widgets/nuxeo-selectivity.js [hunk 2]
```diff
@@ -6953,7 +6951,7 @@ input[type='text'].selectivity-multiple-input:focus {
 
 #input:not([readonly]) .selectivity-single-select {
   border-bottom: 1px solid #3a3a54;
-  border-radius: 2px;
+  min-height: calc(2em + 4px);
```
Comment (Gabez0r) [inline ui/widgets/nuxeo-selectivity.js]: Isn't this reductant given that we already have the same rule on `.selectivity-single-select`? 🤔

**Comment by Gabez0r** [comment]
👍

**Comment by nmpcunha** [comment]
Please ensure that pipeline runs smoothly.

**Comment by nmpcunha** [comment]
👍

**Comment by semisse** [comment]
Test and Push: https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-sfialho/11/

**Pull request merged**